legislators-current.yaml

```diff
@@ -10495,7 +10495,7 @@
     url: https://www.hoeven.senate.gov
     address: 338 Russell Senate Office Building Washington DC 20510
     phone: 202-224-2551
-    contact_form: https://www.hoeven.senate.gov/contact/email-the-senator/form
+    contact_form: http://www.hoeven.senate.gov/public/index.cfm/email-the-senator
```

Comment:
This is for a previous term so unless you know that the URL we had was incorrect at the time (i.e. January 2017), this should be put back.

Reply:
@JoshData Thanks for reviewing! I believe this was corrected in my second commit. The first commit did indeed change the `contact_form` for a previous term.

Reply:
No worries!